aspnetcore/publishing/visual-studio-tools-for-docker.md

```diff
@@ -1,127 +1,134 @@
 
-![Project Add Docker Support](./visual-studio-tools-for-docker/_static/project-add-docker-support.png)
+### New app
 
-*Project Context Menu*
+When creating a new app with the **ASP.NET Core Web Application** project templates, select the **Enable Docker Support** checkbox:
 
-![Right-Click Add Docker Support](./visual-studio-tools-for-docker/_static/right-click-add-docker-support.png)
+![Enable Docker Support checkbox](./visual-studio-tools-for-docker/_static/enable-docker-support-checkbox.png)
 
-When you add Docker support to your project, you can choose either Windows or Linux containers. (The Docker host must be running the same container type. If you need change the container type in the running Docker instance, right-click the **Docker** icon in the System Tray, and choose **Switch to Windows containers** or **Switch to Linux containers**.) 
+If your target framework is .NET Core, the **OS** drop-down allows for the selection of a container type.
 
-The following files are added to the project:
+### Existing app
 
-- **Dockerfile**: the Docker file for ASP.NET Core applications is based on the [microsoft/aspnetcore](https://hub.docker.com/r/microsoft/aspnetcore) image. This image includes the ASP.NET Core NuGet packages, which have been pre-jitted improving startup performance. When building .NET Core Console Applications, the Dockerfile FROM will reference the most recent [microsoft/dotnet](https://hub.docker.com/r/microsoft/dotnet) image.   
-- **docker-compose.yml**: base Docker Compose file used to define the collection of images to be built and run with docker-compose build/run.   
-- **docker-compose.dev.debug.yml**: additional docker-compose file with for iterative changes when your configuration is set to debug. Visual Studio will call -f docker-compose.yml -f docker-compose.dev.debug.yml to merge these together. This compose file is used by Visual Studio development tools.   
-- **docker-compose.dev.release.yml**: additional Docker Compose file to debug your release definition. It will volume mount the debugger so it doesn't change the contents of the production image.  
+The Visual Tools for Docker don't support adding Docker to an existing ASP.NET Core project targeting .NET Framework. For ASP.NET Core projects targeting .NET Core, there are two options for adding Docker support via the tooling. Open the project in Visual Studio, and choose one of the following options:
 
-The *docker-compose.yml* file contains the name of the image that is created when project is run. 
+- Select **Docker Support** from the **Project** menu.
+- Right-click the project in Solution Explorer and select **Add** > **Docker Support**.
 
-```
-version '2'
+## Docker assets overview
+
+The Visual Studio Tools for Docker add a *docker-compose* project to the solution. It contains the following:
```

Comment:
add > adds

Reply:
> The Visual Studio Tools for Docker add a *docker-compose* project to the solution. It contains the following:

The Visual Studio Tools for Docker add a *docker-compose* project to the solution, containing the following: